tests/p2p/test_protocol.py

```diff
@@ -171,12 +169,6 @@ def test_invalid_peer_id(self):
         yield self._send_cmd(self.conn.proto1, 'PEER-ID', json.dumps(invalid_payload))
         self._check_result_only_cmd(self.conn.peek_tr1_value(), b'ERROR')
         self.assertTrue(self.conn.tr1.disconnecting)
-        # When a DNS request is made to twisted client, it starts a callLater to check the resolv file every minute
```

Comment:
No need to do this anymore?

Reply:
This code is really old code. Later we added a proper clean up in the `tearDown()` of the parent class.